tests/core/test_indexset.py

```diff
@@ -90,8 +91,6 @@ def test_add_elements(self, test_mp, request):
     def test_list_indexsets(self, test_mp, request):
         test_mp = request.getfixturevalue(test_mp)
         run = test_mp.runs.create("Model", "Scenario")
-        # Per default, list() lists only `default` version runs:
-        run.set_as_default()
         indexset_1 = run.optimization.indexsets.create("Indexset 1")
```

Comment:
Might be useful to put this index set-creation into a utility function to avoid duplication of code?

Reply:
The idea sounds good and I implemented it, but dealing with the new function actually takes more lines than before in some cases. In particular, the function uses the backend layer to maximize speed, but the core layers sometimes need the core objects returned instead of abstract/backend ones, so they need to be converted.

Reply:
Ok, thanks for trying

Reply:
I think the solution we now have is actually fine and I would also bring that to the other branches. Or do you want me to revert that change?

Reply:
Sorry, I misunderstood your response, it thought that you decided against actually using this because it used more lines of code...